lib/Common/PlatformAgnostic/Numbers.h

```diff
@@ -0,0 +1,51 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+#ifndef RUNTIME_PLATFORM_AGNOSTIC_COMMON_NUMBERS
+#define RUNTIME_PLATFORM_AGNOSTIC_COMMON_NUMBERS
+
+namespace PlatformAgnostic
+{
+namespace Numbers
+{
+    class Utility
+    {
+        class NumbersLocale
+        {
+            WCHAR localeThousands;
+            WCHAR localeDecimal;
+            WCHAR localeNegativeSign;
+
+            int maxDigitsAfterDecimals;
+            WCHAR defaultDecimalDot;
+            WCHAR defaultDecimalComma;
+
+        public:
+
+            NumbersLocale();
+
+            inline int   GetMaxDigitsAfterDecimals() { return maxDigitsAfterDecimals; }
+            inline WCHAR GetLocaleThousands()        { return localeThousands; }
+            inline bool  HasLocaleThousands()        { return localeThousands != 0; }
+            inline WCHAR GetLocaleDecimal()          { return localeDecimal; }
+            inline WCHAR GetLocaleNegativeSign()     { return localeNegativeSign; }
+
+            inline bool  IsDecimalPoint(const WCHAR wc)
+                                                     { return wc == defaultDecimalDot
+                                                           || wc == defaultDecimalComma; }
+        };
+
+        static NumbersLocale numbersLocale;
```

Comment:
Sure it can but I'm not sure if we should consider them during the same process lifecycle?